WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackFeaturesRemovalCoordinator.swift

```diff
@@ -108,6 +108,7 @@ class JetpackFeaturesRemovalCoordinator {
     }
 
     /// Used to determine if the Jetpack features should be removed based on the removal phase.
+    @objc
```

Comment:
Regarding `JetpackFeaturesRemovalCoordinator.shouldRemoveJetpackFeatures()`
I've removed this function in #19817 based on this discussion https://github.com/wordpress-mobile/WordPress-iOS/pull/19816#discussion_r1058906631 and replaced it with `JetpackFeaturesRemovalCoordinator. jetpackFeaturesEnabled()`

Let me know if you need help addressing this conflict or if you think we should re-add `shouldRemoveJetpackFeatures` 👍

Reply:
Ok no problem, I'll update this PR with that change to solve the conflicts.

> Let me know if you need help addressing this conflict or if you think we should re-add shouldRemoveJetpackFeatures 👍

Thanks! Let's stick with the new function `JetpackFeaturesRemovalCoordinator.jetpackFeaturesEnabled()`, it makes more sense as for most of the conditions I had to negate the value when using `shouldRemoveJetpackFeatures()` 😅 .

Reply:
Conflicts solved in https://github.com/wordpress-mobile/WordPress-iOS/pull/19823/commits/6336f6260ae944790b94a5be202bb71563ffcd9c.